Persistant-Storage-Snapshot/DeserializationEx.java

```diff
@@ -0,0 +1,35 @@
+// where should this go? same folder as ServerMain? as methods in Snapshotter or ServerMain?
+
+import codeu.chat.server.Model;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+
+public class DeserializationEx {
+
+	public void deserialize() {
+		try {
+			FileInputStream fs = new FileInputStream("StoredData.txt");
+			ObjectInputStream os = new ObjectInputStream(fs);
+
+			// saving the objects that are serialized
+			// Is there only one object (the data) that needs to be serialized?
+			Object one = os.readObject();
+
+			// Cast the object back to Model
+			Model data = (Model) one;
```

Comment:
I would probably return it and then you can use it. It looks like the Model is setup in Server.java here:

https://github.com/lianasyrkett/CodeU-Summer-2017/blob/da1993fab6e3aebd4d93e3ceb6bd46359d4dbd2b/src/codeu/chat/server/Server.java#L61

You basically need to initialize this variable in the Server constructor instead, but have it optionally deserialize it instead of creating a new one if there is a file path flag set.